README.md

```diff
@@ -4,7 +4,131 @@
 - [GoDoc](https://godoc.org/go.linecorp.com/centraldogma)
 - [CLI manual](https://line.github.io/centraldogma/client-cli.html)
 
-## How to build
+## Usage
+
+### Getting started
+
+```go
+import "go.linecorp.com/centraldogma"
+
+// create client with OAuth2 token
```

Comment:
I think `issued in the Web UI` is a little bit complex here.

Because user use the library if and only if they knew about central dogma, they also installed and used it. So I think we don't need to mention here.

Reply:
Yeah, we can remove the sentence. Just the link is enough. :)